articles/azure-app-configuration/concept-enable-rbac.md

```diff
@@ -33,5 +33,8 @@ Azure provides the following built-in RBAC roles for authorizing access to App C
 - **Contributor**: Use this role to manage the App Configuration resource. While the App Configuration data can be accessed using access keys, this role does not grant access to the data using Azure AD.
 - **Reader**: Use this role to give read access to the App Configuration resource. This does not grant access to the resource's access keys, nor to the data stored in App Configuration.
 
+> [!NOTE]
+> The Azure portal and CLI supports only HMAC authentication to access App Configuration data currently. Therefore, users of the Azure portal and CLI require the Contributor role to access the access keys of the App Configuration resource.
```

Comment:
Recommend for the first line: "Currently, the Azure portal and CLI only support HMAC authentication to access App Configuration data. AAD authentication is not supported."

Also, "access the access keys" could be changed to "retrieve the access keys"

Reply:
Also, just to further reinforce this point, you could add a line at the end, something like: "Granting `App Configuration Data Reader` or `App Configuration Data Owner` permissions has no impact on access through the portal and CLI."

Reply:
Good suggestions. Updated.